### PR TITLE
First version, lots of text and still a basic description

### PR DESCRIPTION
**Pull request by lvarin**
## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.rahtiapp.fi/origin/rahti-migration/cloud/rahti/rahti-migration/) of your branch.

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/cloud/rahti/rahti-migration.md [hunk 1]
```diff
@@ -1,26 +1,53 @@
...
 
-## How to log in Rahti 2?
+## General steps
+
+Before you start the migration, you need to gather information about your application:
+
+1. Where is the **data stored**? And how is it **accessed**? Do you use a database?
+     1. If you use a PostgreSQL database hosted in Rahti, think about migrating to [Pukki DBaaS](../../dbaas/).
+     1. If you use Read-Write-Once (RWO) volumes, you can easily migrate them to Rahti 2. Just follow the instruction in the [How to use storage?](#how-to-use-storage) section.
+     1. If you use Read-Write-Many (RWX), you have to check why are using it. It may be 3 two main options: (1) It was the default and you are not really mounting the same volume in several Pod, or (2) You need to moiunt the same volume in several Pods. If you are in option (2), sadly there in not yet a supported solution in Rahti 2 for RWX, please contact us at <servicedesk@csc.fi> about your use case, we are gathering customer needs to better develop the RWX solution.
```
Comment (tolonenj) [inline docs/cloud/rahti/rahti-migration.md:16]: typo in word mount (moiunt)

"sadly there in not" -> "there is not"

**Review by tolonenj — CHANGES_REQUESTED** [review]
Thank you Álvaro - I think these are good, reasonable, and well documented steps to take within the migration.

File: docs/cloud/rahti/rahti-migration.md [hunk 2]
```diff
@@ -13,15 +13,18 @@ Before you start the migration, you need to gather information about your applic
...
-     1. If you use Read-Write-Many (RWX), you have to check why are using it. It may be 3 two main options: (1) It was the default and you are not really mounting the same volume in several Pod, or (2) You need to moiunt the same volume in several Pods. If you are in option (2), sadly there in not yet a supported solution in Rahti 2 for RWX, please contact us at <servicedesk@csc.fi> about your use case, we are gathering customer needs to better develop the RWX solution.
+     1. If you use Read-Write-Many (RWX), you have to check why are you using it. It may be two main options: (1) It was the default and you could use a RXO volume instead because you are only mounting the volume once, or (2) You need to mount iat the same time the same volume in several Pods. If you are in option (2), sadly there in not yet a supported solution in Rahti 2 for RWX, please contact us at <servicedesk@csc.fi> and let us know your use case, we are gathering customer needs to better develop the RWX solution. And we will work together on possible alternatives.
```
Comment (tolonenj) [inline docs/cloud/rahti/rahti-migration.md]: "mount iat" -> "mount at"

I like the way how you word the RWX 👍

File: docs/cloud/rahti/rahti-migration.md [hunk 3]
```diff
@@ -105,7 +108,10 @@ In Rahti 2 the default limits are lower than the default quota:
...
+
+!!! info "Why are limits tighter?"
+    Rahti 1 resource range (difference between request and limits) was too wide. This made the scheduler's job harder, as every Pod looked the same regarding resource needs (every Pod requested the same resources). This increased the "noisy neighbours effect", were Pods hungry for resources were placed on the same nodes as more modest Pods. The hungry ones were starving the more modest ones. With tigher limits and a maximum faxctor of between request and limit, Pods will need to be configured with more explicit limits.
```
Comment (tolonenj) [inline docs/cloud/rahti/rahti-migration.md:114]: "tigher" -> "tighter"
"faxctor" -> "factor"

**Review by tolonenj — COMMENTED** [review]
A couple of typos - but I liked the additions! They made the article more coherent to read.

File: docs/cloud/rahti/rahti-migration.md [hunk 4]
```diff
@@ -1,26 +1,56 @@
...
 
-## How to log in Rahti 2?
+## General steps
+
+Before you start the migration, you need to gather information about your application:
+
+1. Where is the **data stored**? And how is it **accessed**? Do you use a database?
+     1. If you use a PostgreSQL database hosted in Rahti, think about migrating to [Pukki DBaaS](../../dbaas/).
+     1. If you use Read-Write-Once (RWO) volumes, you can easily migrate them to Rahti 2. Just follow the instruction in the [How to use storage?](#how-to-use-storage) section.
+     1. If you use Read-Write-Many (RWX), you have to check why are you using it. It may be two main options: (1) It was the default and you could use a RXO volume instead because you are only mounting the volume once, or (2) You need to mount at the same time the same volume in several Pods. If you are in option (2), sadly there is not yet a supported solution in Rahti 2 for RWX, please contact us at <servicedesk@csc.fi> and let us know your use case, we are gathering customer needs to better develop the RWX solution. And we will work together on possible alternatives.
```
Comment (trispera) [inline docs/cloud/rahti/rahti-migration.md:16]: `...It was the default and you could use a RXO volume...`
I think you meant `RWO` instead of `RXO`?

**Comment by trispera** [comment]
LGTM. One comment otherwise good additions and explanations.

File: docs/cloud/rahti/rahti-migration.md (hunk 1 above)
Comment (lvarin) [reply docs/cloud/rahti/rahti-migration.md]: fixed

File: docs/cloud/rahti/rahti-migration.md (hunk 2 above)
Comment (lvarin) [reply docs/cloud/rahti/rahti-migration.md]: fixed

File: docs/cloud/rahti/rahti-migration.md (hunk 3 above)
Comment (lvarin) [reply docs/cloud/rahti/rahti-migration.md:114]: fixed

**Comment by lvarin** [comment]
All typos fixed

**Review by trispera — APPROVED** [review]
LGTM

**Review by tolonenj — APPROVED** [review]

**Pull request merged**